README.rst

```diff
@@ -93,8 +93,8 @@ For more information on contributing to SunPy, please read our `Newcomers' guide
 .. _Developers Guide: https://docs.sunpy.org/en/latest/dev_guide/index.html
 .. _`#sunpy:openastronomy.org`: https://app.element.io/#/room/#sunpy:openastronomy.org
 .. _issues page: https://github.com/sunpy/sunpy/issues
-.. _Newcomers' guide: https://docs.sunpy.org/en/latest/dev_guide/newcomers.html
-.. _Development installation:  https://docs.sunpy.org/en/latest/dev_guide/newcomers.html#setting-up-a-development-environment
+.. _Newcomers' guide: https://docs.sunpy.org/en/latest/dev_guide/contents/newcomers.html
```

Comment:
This was an oversight when we made changes to our "dev_guide", I need to make sure other links are not broken.

Reply:
I did a rough grep through the repository and only found one more link that needs updated in the dev_guide (just committed that change now). It could be replaced with a relative link to the page but I guess it is best to give an absolute link to the latest developer documentation instead.

Reply:
Yeah there are some issues with how we build our dev docs, a direct URL is better.